op-conductor/conductor/service_test.go

```diff
@@ -879,3 +880,115 @@ func (s *OpConductorTestSuite) TestHandleInitError() {
 func TestControlLoop(t *testing.T) {
 	suite.Run(t, new(OpConductorTestSuite))
 }
+
+// TestRollupBoostEnableDisable tests basic enabling and disabling of Rollup boost
+// during sequencer start and stop operations.
+func (s *OpConductorTestSuite) TestRollupBoostEnableDisable() {
```

Comment:
please write tests with synchronization mode enabled, and verify internal states step by step.

This is the most critical part of the control and we need it be well tested.

Please also cover function calls to `OverrideLeader` as well

Reply:
Current test does not fully cover the failure situations (like the above bug I mentioned)

Reply:
The scenarios I'm thinking are at least:
1. rollupboost enable success / failure
2. rollupboost disable success / failure
4. override leader works with (rollupboost call success / fail)